optuna/distributions.py

```diff
@@ -48,6 +48,36 @@ def to_internal_repr(self, param_value_in_external_repr):
 
         return param_value_in_external_repr
 
+    @abc.abstractmethod
+    def empty(self):
+        # type: () -> bool
+        """Test whether the range of this distribution is empty.
+
+        When this method returns :obj:`True`, samplers cannot sample any value
```

Comment:
Could you add a link to `optuna.samplers`?

Reply:
Added: https://github.com/pfnet/optuna/pull/431/commits/64d9ca4d0008f8891a0fc48d2c8aa3b20aeb1dbe